config.xml

```diff
@@ -16,7 +16,7 @@
     <preference name="GOOGLE_MAPS_IOS_API_KEY" value="" />
     <preference name="ScrollEnabled" value="false" />
     <preference name="AndroidPersistentFileLocation" value="Compatibility" />
-    <preference name="android-minSdkVersion" value="21" />
+    <preference name="android-minSdkVersion" value="28" />
```

Comment:
This is going to cause a lot of devices to be unusable - Are we able to use minSDKVersion=21 or 23 instead?